src/features/tutorials/components/GenericTutorial.tsx

```diff
@@ -0,0 +1,113 @@
+import { t } from '@lingui/macro'
+import { useNavigation } from '@react-navigation/native'
+import LottieView from 'lottie-react-native'
+import AnimatedLottieView from 'lottie-react-native'
+import React, { FunctionComponent, useEffect, useRef } from 'react'
+import { Alert } from 'react-native'
+import styled from 'styled-components/native'
+
+import { UseNavigationType } from 'features/navigation/RootNavigator'
+import { _ } from 'libs/i18n'
+import { AnimationObject } from 'ui/animations/type'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiaryGreyDark } from 'ui/components/buttons/ButtonTertiaryGreyDark'
+import { StepDots } from 'ui/components/StepDots'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+type Props = {
+  animation: AnimationObject
+  animationSize: number
+  endFrame?: number
+  title: string
+  subTitle: string
+  text: string
+}
+
+export const GenericTutorial: FunctionComponent<Props> = (props) => {
+  const { navigate } = useNavigation<UseNavigationType>()
+
+  const animationRef = useRef<AnimatedLottieView>(null)
+
+  useEffect(() => {
+    if (props.endFrame) {
+      animationRef.current?.play(0, 62)
+    }
+  }, [])
+
+  function goToHomeWithoutModal() {
+    navigate('TabNavigator')
+  }
+
+  function goToNextTutorial() {
+    Alert.alert('TODO: PC-5960')
+  }
+
+  return (
+    <Container>
+      <Spacer.Flex flex={1} />
+      <Container2>
```

Comment:
Je pense que tu peux appeler cela `Header`:
```suggestion
      <Header>
```

Reply:
Oui carrément, j'ai oublié lors de mon refacto !